components/Players/VideoPlayer.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <v-container fill-height fluid class="pa-0 justify-center">
-    <div ref="videoContainer">
+    <div
+      ref="videoContainer"
+      style="height: 100%; width: 100%; overflow: hidden"
```

Comment:
Could we use a class for this, please?

Also, wouldn't `overflow: hidden` cut part of the video in some cases? (mini-player for example, since you had a scrollbar before, I'm assuming the video was too big for the dialog)

Reply:
We might also just be able to remove this `div` and use the `v-container` instead, transferring the props from this one to its parent.

Reply:
> Could you also reword the commit to better respect the format used elsewhere?
> 
> The context (`sd_small` in your commit) should be the component you edited, in kebab case. So here, a good commit message would be `fix(video-player): video container does not scale` or something similar.

Sure, my bad.

> Could we use a class for this, please?

I tried it the first time and probably forgot to refresh properly because it didn't work. I'll change it to an id

> Also, wouldn't overflow: hidden cut part of the video in some cases? (mini-player for example, since you had a scrollbar before, I'm assuming the video was too big for the dialog)

I don't think so. I tested it with a few files again now, and as seen in the image below it seems like it only exposes some padding bellow.
![image](https://user-images.githubusercontent.com/20001253/104825114-13f32400-5858-11eb-9a94-2af5d564fbd8.png)

Reply:
> We might also just be able to remove this `div` and use the `v-container` instead, transferring the props from this one to its parent.

Just tried this and it worked flawlessly! I will push the change

Reply:
Screenshots:

**SD**
![image](https://user-images.githubusercontent.com/20001253/104827304-777f5080-585c-11eb-97ac-13173fd0f850.png)

**HD**
![image](https://user-images.githubusercontent.com/20001253/104827386-5703c600-585d-11eb-9cb0-293411d517c8.png)


**PiP**
![image](https://user-images.githubusercontent.com/20001253/104827331-bd3c1900-585c-11eb-90cb-52c7e45717b5.png)